src/Illuminate/Collections/Traits/TransformsToResourceCollection.php

```diff
@@ -3,7 +3,6 @@
 namespace Illuminate\Support\Traits;
 
 use Illuminate\Database\Eloquent\Model;
-use Illuminate\Http\Resources\Json\JsonResource;
 use Illuminate\Http\Resources\Json\ResourceCollection;
```

Comment:
Shouldn't `ResourceCollection` also be removed?

Reply:
![CleanShot 2025-04-07 at 15 43 27](https://github.com/user-attachments/assets/c27f96d9-0b4f-4d0e-8a90-008821ada3a7)

There's wouldn't be an issue unless you are explicitly calling `toResource()` method, and if you do you definitely need to require `illuminate/http`